dali/operators/reader/loader/video_loader.h

```diff
@@ -163,13 +163,19 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
       codec_id_(0),
       skip_vfr_check_(spec.GetArgument<bool>("skip_vfr_check")),
       file_list_frame_num_(spec.GetArgument<bool>("file_list_frame_num")),
+      file_list_include_preceding_frame_(spec.GetArgument<bool>(
+                                                          "file_list_include_preceding_frame")),
       pad_sequences_(spec.GetArgument<bool>("pad_sequences")),
       stats_({0, 0, 0, 0, 0}),
       current_frame_idx_(-1),
       stop_(false) {
     DALI_ENFORCE(stride_ > 0, "Stride should be > 0");
     if (step_ < 0)
       step_ = count_ * stride_;
+    if (!file_list_include_preceding_frame_) {
+      DALI_WARN("``file_list_include_preceding_frame_`` is set to False (or not set at all). In "
+                "future releases, the default behavior would be changed to True..");
```

Comment:
```suggestion
                "future releases, the default behavior would be changed to True.");
```

Reply:
Can we guestimate when we will do it, and put it here?

Reply:
I don know tbh. And I prefer not to make promises, if we are not sure.